js/src/index.js

```diff
@@ -27,5 +29,7 @@ window.jsmodule = {
     MultiSlider,
     Tree,
     ResizeSensor,
+    PanelStack,
   },
+  "@/shiny.blueprint/handlers": handlers,
```

Comment:
If you have little time left: let's name it `panelStackHandlers` and place it along the other stuff in `@/shiny.blueprint`.

If you have some more time left: Instead of directly exposing `handlers`, consider adding `openPanel` and `closePanel` functions to `@/shiny.blueprint`. Let them take the necessary arguments (like namespace, panel id, ...). These functions could access an unexported global variable, where `PanelStack` components could register themselves.

Reply:
Little time version applied